client/src/components/FileUploadComponent/FileUpload.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import FileUpload from "./FileUpload"
+
+
+storiesOf("File Upload", module)
+  .add("Default", () =>
+    <FileUpload>
+    </FileUpload>)
+  .add("New User Upload", () =>
+    <FileUpload
+      context="userPhoto"
+      buttonLabel="Browse Files">
```

Comment:
I think you'll need more props than this. We'll want a way to control the file type through a prop, a way to control the header text, the button label, and a callback that can accept the file and do something with it